lib/signs_ui/signs/expiration.ex

```diff
@@ -0,0 +1,86 @@
+defmodule SignsUI.Signs.Expiration do
+  @moduledoc """
+  Handle cleaning up settings in the configuration that have since expired
+  """
+  use GenServer
+
+  @type state :: %{
+          time_fetcher: (() -> DateTime.t()),
+          loop_ms: integer(),
+          sign_state_server: GenServer.server()
+        }
+
+  @spec start_link(Keyword.t()) :: GenServer.on_start()
+  def start_link(opts \\ []) do
+    name = Keyword.get(opts, :name) || __MODULE__
```

Comment:
I think for all these it makes sense to make the default the optional third argument, rather than the `|| ...` construction.

Reply:
Done